shell/common/shell_unittests.cc

```diff
@@ -201,7 +204,8 @@ TEST_F(ShellTest, SecondaryIsolateBindingsAreSetupViaShellSettings) {
 
 TEST(ShellTestNoFixture, EnableMirrorsIsWhitelisted) {
   if (DartVM::IsRunningPrecompiledCode()) {
-    // This covers profile and release modes which use AOT (where this flag does
+    // This covers profile and release modes which use AOT (where this flag
+    does
```

Comment:
Looks like there are some unintentional reformattings in this file

Reply:
Yeah clang format reflowed some comments that I then uncommented.  Will fix